dev-libs/libqt5pas/libqt5pas-1.2.9.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+LAZRELEASE="2.2.4"
+
+if [[ ${PV} = 9999* ]]; then
+	EGIT_REPO_URI="https://gitlab.com/freepascal.org/lazarus/lazarus.git"
+	inherit git-r3
+else
+	SRC_URI="https://sourceforge.net/projects/lazarus/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${LAZRELEASE}/lazarus-${LAZRELEASE}-0.tar.gz/download -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Free Pascal Qt5 bindings library updated by lazarus IDE."
+HOMEPAGE="https://gitlab.com/freepascal.org/lazarus/lazarus"
+LICENSE="LGPL-3"
+SLOT="0/2.2"
+
+DEPEND="
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtx11extras:5
+"
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
+S="${WORKDIR}/lazarus/lcl/interfaces/qt5/cbindings"
+
+src_unpack () {
+	if [[ ${PV} = 9999* ]]; then
+		git-r3_fetch ${EGIT_REPO_URI}
+		git-r3_checkout ${EGIT_REPO_URI} "${WORKDIR}/lazarus" "" "lcl/interfaces/qt5/cbindings"
+	else
```

Comment:
Same, delete as you aren't using a live ebuild

Reply:
I should probably remove src_unpack() altogether then, no?